frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -72,6 +71,32 @@ <h5 class="fw-light">Challenge Settings</h5>
       </div>
     </div>
 
+    <!-- Evaluations and Annoatations -->
+    <div class="row settings-section">
+      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-light fs-18">Evaluation and Annoatations:</span>
+      </div>
+      <div class="ev-action">
+        <div *ngIf="isChallengeHost" class="col-sm-1.5 col-xs-1.5 ev-upload-action">
+          <!-- Evaluations Criteria Description -->
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Evaluation Criteria </strong>
```

Comment:
Can you show these options in next row. Also please use a bigger font for the text.

Reply:
We are using the same font as we use for the `Edit Challenge details`. Should I increase the font size?

Reply:
I see, let's not change the font then. We can change style of the page once we have all the edit features on this page. It doesn't look good right now.